code/common/api.q

```diff
@@ -72,6 +72,21 @@ search:{[s;c]
 // search function
 s:.api.search[;0b]
 
+// input list of namespaces for exportconfig
+KEYS:` sv'`,'key[`]except`$'.Q.an;
+
+// export the current state of config variables, takes in list of symbols of namespaces (e.g. `.usage`.procs)
+exportconfig:{
+  // selects only variables in inpoutted namespaces
+  :?[.api.f`;
+    ((=;`vartype;enlist`variable);(in;`namespace;enlist x));
+    {x!x}enlist`name;
+  // returns name, value and description
+    `val`descrip!((value';`name);`descrip)
+   ];
+ }
+
```

Comment:
Looks good, though can you also add exportallconfig as 

exportallconfig:{exportconfig[torqnamespaces]}